deepmd/entrypoints/train.py

```diff
@@ -210,28 +214,34 @@ def get_modifier(modi_data=None):
     return modifier
 
 
-def get_rcut(jdata):
-    descrpt_data = jdata['model']['descriptor']
+def parse_rcut(descrpt_data):
     rcut_list = []
     if descrpt_data['type'] == 'hybrid':
         for ii in descrpt_data['list']:
             rcut_list.append(ii['rcut'])
     else:
         rcut_list.append(descrpt_data['rcut'])
+    return rcut_list
+
+def get_rcut(jdata):
+    descrpt_data = jdata['model']['descriptor']
+    rcut_list = []
+    rcut_list.extend(parse_rcut(descrpt_data))
```

Comment:
I do not think you need to revise the `get_rcut`.

Reply:
if I don't split get_rcut into two function, I can't reuse it in the train_mt